docs/syncing-forks.md

```diff
@@ -94,6 +94,8 @@ This is off by default and you'd need to enable `Allow merge commits` in the rep
 
 ![allow-merge-commit](./assets/allow-merge-commit.png)
 
+If merge queue is on, you will also need to disable it since merge queue uses squash and merge by default. Merge queue can be turned off in Settings => Rules => Rulesets => main => Require Merge Queue.
```

Comment:
```suggestion
Select "Pull request title and description" as the default commit message so that merge commits use conventional commit format

If merge queue is on, you will also need to disable it since merge queue uses squash and merge by default. Merge queue can be turned off in Settings => Rules => Rulesets => main => Require Merge Queue.
```